src/open_inwoner/scss/components/Notification/_Notification.scss

```diff
@@ -25,6 +26,14 @@
     --notification-color-text: var(--color-success);
   }
 
+  &--compact {
+    margin: var(--spacing-medium) 0;
+
+    p {
```

Comment:
IIRC there was something about our CSS code-style where we don't use tag names in selectors (hence the `<p class="p">` everywhere, but I'm not sure. I think it might have to do with specificity of the selector? Maybe @svenvandescheur has ideas about this.